cves/kernel/CVE-2016-9685.yml

```diff
@@ -55,7 +55,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The vulnerability is multiple memory leaks in the xfs mudule of the linux 
+  filesystem (fs/xfs/xfs_attr_list.c) which could enable a local user to perform 
+  a Denial of Service attack.
```

Comment:
Good start, but let's expand upon this. Your audience won't know what xfs is and what a module is in it - research that and explain. Explain why it's the attributes list - that seems interesting. And then explain some more about what a local user doing a DOS would do. 

Remember that your target audience is SE sophomores - so they'll know a little bit about memory leaks, but not much more beyond that.

Reply:
Will do 👍  Thank you for the great suggestions!